### DISCUSSION TITLE
Release v1.0.13 / 5.55.13

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1671] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixed a security issue.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Fixed
- FIXED SECURITY ISSUE: Hard link creation was not properly filtered (thanks Diversenok)
- fixed issue with checking the certificate entry.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.13'>Release v1.0.13 / 5.55.13</a>.</em>

**Comment by BilleBarrett** [comment]
Tried to look at passwords in Chrome and got these errors
07:39:55.330   Installation Directory: C:\Program Files\Sandboxie-Plus                                                       
07:39:55.330   Sandboxie-Plus Version: 1.0.13 (5.55.0)                                                                       
07:39:55.330   Loaded Config: C:\WINDOWS\Sandboxie.ini                                                                       
07:39:55.330   Data Directory: C:\Users\BilleB\AppData\Local\Sandboxie-Plus                                                  
07:39:55.449   Sandboxie config has been reloaded                                                                            
08:15:36.168   CredentialUIBroker.exe (13592): SBIE2224 Sandboxed program has crashed: CredentialUIBroker.exe [DefaultBox]   
08:16:05.499   CredentialUIBroker.exe (12564): SBIE2224 Sandboxed program has crashed: CredentialUIBroker.exe [DefaultBox]   
08:16:37.724   CredentialUIBroker.exe (12224): SBIE2224 Sandboxed program has crashed: CredentialUIBroker.exe [DefaultBox]   
08:18:36.316   CredentialUIBroker.exe (16288): SBIE2224 Sandboxed program has crashed: CredentialUIBroker.exe [DefaultBox]   
Had ro leave Sandboxie-Plus to get them
Chrome is up to date Version 99.0.4844.51 (Official Build) (64-bit)
Thanks
Bill

**Comment by BilleBarrett** [comment]
Just had a crash, where do I send the DMP file

**Reply by isaak654** [reply]
I would suggest to open a new problem report, in order to be tracked easily:
https://github.com/sandboxie-plus/Sandboxie/issues/new?assignees=&labels=&template=1-problem_report.yml

**Reply by BilleBarrett** [reply]
Let me ask this, my client has been having a problem doing Zoom meetings, his video doe not work
I ran the zoom meeting outside of Sandboxie Classic and everything runs OK
Should I upgrade him to the Plus version and try it that way?
Thanks
Bill

**Reply by isaak654** [reply]
I don't recall a particular difference between Sbie Classic and Plus regarding software compatibility... but with Plus you can switch a number of isolation options easily than Classic UI.

**Comment by jarekt** [comment]
Is it just me or is the [download page](https://sandboxie-plus.com/downloads/) still pointing to the 1.0.10 version?

**Reply by isaak654** [reply]
See #1682

**Comment by BilleBarrett** [comment]
I always use sandboxie-plus/Sandboxie <https://github.com/sandboxie-plus/Sandboxie>  for updtes

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.***
Is your computer up to date and secure “NO” lets talk
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
